models/shipping-method/src/shipping-method/generator.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  ClientLogging,
+  LocalizedString,
+  Reference,
+} from '@commercetools-test-data/commons';
+import { sequence, fake, Generator } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import { TShippingMethod } from './types';
+
+// https://docs.commercetools.com/api/projects/shippingMethods#shippingmethod
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+const generator = Generator<TShippingMethod>({
+  fields: {
+    id: fake((f) => f.datatype.uuid()),
+    version: sequence(),
+    key: fake((f) => f.lorem.slug(2)),
+    name: fake((f) => f.lorem.words(2)),
+    localizedName: fake((f) => LocalizedString.random()),
+    localizedDescription: fake(() => LocalizedString.random()),
+    taxCategory: fake(() => Reference.random().typeId('tax-category')),
```

Comment:
This should be a full blown tax category and then in the transformer for rest turned into a reference based on the id of this field and for graphql the `taxCategoryRef` field is built based on the id.

Reply:
Missed that update! Thank you 🙏